torch/onnx/symbolic_helper.py

```diff
@@ -163,33 +165,41 @@ def _parse_arg(
     )
 
 
+@_beartype.beartype
 def _node_get(node: _C.Node, key: str):
     """Gets attributes of a node which is polymorphic over return type."""
     assert isinstance(node, _C.Node)
     sel = node.kindOf(key)
     return getattr(node, sel)(key)
 
 
+@_beartype.beartype
 def _is_onnx_constant(value: _C.Value):
     """Whether a Value is an ONNX constant."""
     return value.node().kind() == "onnx::Constant"
 
 
-def _maybe_get_const(value: _C.Value, descriptor: _ValueDescriptor):
+@_beartype.beartype
+def _maybe_get_const(
+    value: Optional[Union[_C.Value, torch.Tensor, Number, Sequence]],
+    descriptor: _ValueDescriptor,
+):
     # NOTE: prim::Constant at this stage usually means something not compatible in ONNX,
     # otherwise it'd be converted to onnx::Constant
-    if _is_value(value) and _is_onnx_constant(value):
+    if isinstance(value, _C.Value) and _is_onnx_constant(value):
```

Comment:
Changed for mypy checks on `_is_onnx_constant`

Reply:
At least add a TODO comment here and on `_is_value` describing the mypy issue for a future fix. Otherwise this will be forgotten

Reply:
Done